scenarios/academic/KDD2020-tutorial/run_transE.sh

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Maybe this can be used and we can find a better place for it? @Leavingseason @miguelgfierro